middlewares/validators/user.js

```diff
@@ -102,6 +102,9 @@ async function getUsers(req, res, next) {
           "string.empty": "size must contain value in range 1-100",
           "string.pattern.base": "size must be in range 1-100",
         }),
+      q: joi.string().optional().messages({
+        "string.empty": "q must not be empty and should have both a key and a value",
```

Comment:
Does q here refer to query?
Since this would be appearing in either logs or in user message, could we keep it more descriptive please

Reply:
Done Added the changes with `query` in the description.